js/components/UI/NovaButton/NovaButton.js

```diff
@@ -22,7 +22,12 @@ NovaButton.propTypes = {
   title: PropTypes.string.isRequired,
   onPressFunc: PropTypes.func,
   color: PropTypes.string.isRequired,
-  onPressParams: PropTypes.object
+  onPressParams: PropTypes.object,
+  style: PropTypes.oneOfType([
```

Comment:
Use `ViewPropStyles.style` as a prop type instead.

Reply:
I'm struggling to find any information about how this might work. No info in react-native docs, so perhaps it's a third-party thing? Any hint or link to a component or docs would be appreciated. Thanks!

Reply:
```
import { ViewPropTypes } from 'react-native';

// ...

MyComponent.propTypes = {
  style: ViewPropTypes.style
};
```

Reply:
^^^ If you get an error when using this on the Touchable component you can leave your code as-is (but please do use this proptype on for the style prop on all Views in the future).

Reply:
Thanks. It works on here, but not NovaH2. Possible because the style prop is used on a `<TouchableHighlight>` here but on `<Text>` in NovaH2. I've pushed up all my commits now, please review.